code/modules/mob/living/carbon/carbon_emote.dm

```diff
@@ -168,15 +168,38 @@
 	if(!.)
 		return
 
-	if(user.l_hand || user.r_hand)
-		return TRUE
+	if(user.get_active_hand() || user.get_inactive_hand())
+		var/obj/item/thing
+
+		if(user.get_active_hand())
+			thing = user.get_active_hand()
+		else
+			thing = user.get_inactive_hand()
+
+		if(thing.flags & ABSTRACT && !istype(thing, /obj/item/grab))
+			to_chat(user, "<span class='warning'>You cannot twirl [thing]!</span>")
```

Comment:
You need to move this check (and specifically, the to_chat) out to `run_emote`. `*help` runs through all emotes and calls `can_run_emote` to see if they can be used, meaning that any to_chats here will be sent when someone tries to use `*help`. If you need to duplicate logic here, it might be a necessary evil

See: what happens if you try to call `*help` on live rn with an empty hand:

![image](https://github.com/ParadiseSS13/Paradise/assets/89928798/e23c0913-9f0a-439f-adad-6ce4181f8ca3)